### ISSUE TITLE
[api-extractor] ".." or "." import paths not correctly resolved

### ISSUE THREAD

**Issue by eelco** [Microsoft/web-build-tools#848]
These both work:
```
import { Foo } from "./"
import { Bar } from "../"
```

These both don’t:
```
import { Foo } from "."
import { Bar } from ".."
```

The error will be
`Error: Program Bug: The symbol Foo is being imported after it was already registered as non-imported`

**Comment by pgonzal** [comment]
@eelco could you take a look at PR https://github.com/Microsoft/web-build-tools/pull/849 ?

My repro is in commit https://github.com/Microsoft/web-build-tools/pull/849/commits/adad81efb8216af39b6a76650ae85652fad40810 .  Although it does produce incorrect \*.d.ts output (which is fixed by this PR) I was not able to reproduce the specific error that you encountered.

If you can't figure out an isolated repro, perhaps you could simply test this PR branch against your real project?  To do that, instead of running this:

```
C:\MyProject> api-extractor run --local
```

...you would do this:

```
C:\MyProject> node C:\web-build-tools\apps\api-extractor\lib\start.js run --local
```

(where `C:\web-build-tools` contains the branch that you compiled using `rush install && rush rebuild`.)

**Comment by eelco** [comment]
Apologies for the late reply! I just verified that the PR works by running it on my project just before I applied the workaround. (And after first reproducing it again with 5.13.0) 👏

**Comment by pgonzal** [comment]
Cool. I will merge&publish this fix today.

**Comment by pgonzal** [comment]
This is fixed in **API Extractor 6.0.3**
